CHANGELOG.md

```diff
@@ -2,8 +2,19 @@
 
 ## [Unreleased][unreleased]
 
+## Added
+
+## Changed
+
+## Fixed
+
+<a id="1_47_1"></a>
+
+## [1.47.1] - 2024-11-01
+
 ## Added
 - Added support for selecting multiple files in the attachment file picker. #4278
+- Several accessibility improvements for keyboard navigation
```

Comment:
There is already a line about this below `improve keyboard and screen-reader accessibility #4210`

Reply:
Ok then I move that up to Added